firebase-vscode/webpack.common.js

```diff
@@ -219,11 +219,11 @@ const extensionConfig = {
   },
 };
 
-function makeWebConfig(entryName) {
+function makeWebConfig(entryName, entryPath = "") {
   return {
     name: entryName,
     mode: "none", // this leaves the source code as close as possible to the original (when packaging we set this to 'production')
-    entry: `./webviews/${entryName}.entry.tsx`,
+    entry: `./webviews/${entryPath}${entryName}.entry.tsx`,
```

Comment:
Right now, this is fragile to mismatched slashes (ie 'myEntryPath/' works, but 'myEntryPath' does not). You probably want to use path.join to construct this..

Reply:
done